backend/pom.xml

```diff
@@ -50,6 +50,21 @@
       <artifactId>commons-cli</artifactId>
       <version>1.3.1</version>
     </dependency>
+    <!--<dependency>-->
```

Comment:
outdated diff. uncommented once jackson dependency issues were fixed. pushed another commit, which was missed in this review